src/main/java/com/ludo/study/studymatchingplatform/study/domain/study/Study.java

```diff
@@ -45,294 +30,339 @@
+        if (this.endDateTime.isBefore(now)) {
+            this.status = StudyStatus.COMPLETED;
+        }
+    }
+
+    public void deactivateForRecruitment() {
+        this.recruitment.softDelete();
+    }
+
+    public void activateForRecruitment() {
+        this.recruitment.activate();
+    }
+
+    public Boolean ensureHasRecruitment() {
+        if (this.recruitment != null && !this.recruitment.isDeleted()) {
+            return Boolean.TRUE;
+        }
+        return Boolean.FALSE;
+    }
+
+    public void update(final String title, final Category category, final Integer participantLimit,
+                       final Way way, final Platform platform, final String platformUrl,
+                       final LocalDateTime startDateTime, final LocalDateTime endDateTime) {
+        if (title != null) {
+            this.title = title;
+        }
+        if (category != null) {
+            this.category = category;
+        }
+        if (participantLimit != null) {
+            this.participantLimit = participantLimit;
+        }
+        if (way != null) {
+            this.way = way;
+        }
+        if (platform != null) {
+            this.platform = platform;
+        }
+        if (platformUrl != null) {
+            this.platformUrl = platformUrl;
+        }
+        if (startDateTime != null) {
+            this.startDateTime = startDateTime;
+        }
+        if (endDateTime != null) {
+            this.endDateTime = endDateTime;
+        }
+    }
+
+    public void ensureReviewPeriodAvailable(final LocalDateTimePicker localDateTimePicker) {
+        final LocalDateTime now = localDateTimePicker.now();
+        final LocalDateTime reviewAvailStartTime = endDateTime.plusDays(3).minusSeconds(1);
+        final LocalDateTime reviewAvailEndTime = endDateTime.plusDays(14).plusSeconds(1);
+
+        // Order Specific
+        if (isReviewPeriodElapsed(now)) {
+            throw new InvalidReviewPeriodException("리뷰 작성 기간이 지났습니다.", reviewAvailStartTime, reviewAvailEndTime);
+        }
+
+        if (isNotReviewPeriodYet(now)) {
+            throw new InvalidReviewPeriodException("아직 리뷰 작성 기간이 되지 않았습니다.", reviewAvailStartTime, reviewAvailEndTime);
+        }
+
+        System.out.println("min avail: " + reviewAvailStartTime.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+        System.out.println("max avail: " + reviewAvailEndTime.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+        System.out.println("now : " + now.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+
+    }
+
+    public boolean isNotReviewPeriodYet(final LocalDateTime now) {
+        final LocalDateTime reviewAvailStartTime = endDateTime.plusDays(3);
+        return now.isBefore(reviewAvailStartTime);
+    }
+
+    public boolean isReviewPeriodElapsed(final LocalDateTime now) {
+        final LocalDateTime reviewAvailEndTime = endDateTime.plusDays(14);
+        return now.isAfter(reviewAvailEndTime);
+    }
```

Comment:
```suggestion
    }
    
    private LocalDateTime getReviewAvailStartTime() {
        return endDateTime;
    }
    
    private LocalDateTime getReviewAvailEndTime() {
        return endDateTime.plusDays(14);
    }
```

날짜 계산 로직이 `ensureReviewPeriodAvailable`, `isNotReviewPeriodYet`, `isReviewPeriodElapsed` 메서드에 흩어져 있는 것으로 생각되어, 메서드 분리를 하는 것이 좋아보입니다 🙂

리뷰 가능 시간에 대한 요구사항이 변경되어, 이를 반영했습니다.